functional-samples/sample.bookmarks/README.md

```diff
@@ -0,0 +1,15 @@
+# Bookmarks Manager
+
+This sample demonstrates how to use the [Bookmarks API](https://developer.chrome.com/docs/extensions/reference/api/bookmarks) to search, add, edit, and delete bookmarks.
```

Comment:
We aren't totally consistent here but I try to avoid coming up with names for things and prefer to just use the name of the API :)

```suggestion
This sample demonstrates how to use the [chrome.bookmarks](https://developer.chrome.com/docs/extensions/reference/api/bookmarks) API to search, add, edit, and delete bookmarks.
```

Reply:
Yes, I agree